tsconfig.json

```diff
@@ -1,77 +1,36 @@
 {
```

Comment:
This is the new root `tsconfig.json` for the whole project. You can see that there isn't too much going on here - here we basically:

- inherit from our base config
- use `files` to include a bunch of files
- list sub project which this main project depends on in `references`